### ISSUE TITLE
sevtech ages of the sky: cannot insert death compass into grindstone

### ISSUE THREAD

**Issue by Anniebe11** [DarkPacks/SevTech-Ages#4299]
## Bug Description
the uses for the death compass in jei shows that it can be ground into bonemeal using either the hand operated or horse powered grindstone however I am unable to put it in to either .

### Did this behavior use to work in the previous version?
as far as I know this is the earliest version

## Steps to Reproduce (for bugs)

1. make and place grindstone
2. die
3. attempt to insert death compass into grindstone

## Client Information
* Modpack Version: unknown, can't work out how to see this in gd launcher
* Java Version: version 8 update 251
* Launcher Used: GD Launcher
* Memory Allocated: 6144
* Server/LAN/Single Player: single player
* Optifine Installed: no
* Shaders Enabled: no

## World Information
* Modpack Version world created in: whatever this version is (to my knowledge the only version released as an alpha)
* Additional Content Installed: none

**Comment by Anniebe11** [comment]
found the version in the manifest
3.2.0-alpha.2-skyblock

**Comment by Om3ga12** [comment]
You need to put the compass thru a crafting grid to clear the information like the death coordinates and what not, then you may grind it.

**Comment by sam-kirby** [comment]
Indeed, I previously suggested that, though there is no indication @Anniebe11 has tried it.

https://github.com/DarkPacks/SevTech-Ages/issues/4298#issuecomment-699146505
